modules/core/include/opencv2/core/hal/intrin_avx512.hpp

```diff
@@ -1473,6 +1473,32 @@ inline v_float64x8 v_cvt_f64(const v_float32x16& a)
 inline v_float64x8 v_cvt_f64_high(const v_float32x16& a)
 { return v_float64x8(_mm512_cvtps_pd(_v512_extract_high(a.val))); }
 
+// from (Mysticial and wim) https://stackoverflow.com/q/41144668
+inline v_float64x8 v_cvt_f64(const v_int64x8& v)
+{
+#if CV_AVX_512DQ
+    return v_float64x8(_mm512_cvtepi64_pd(v.val));
+#else
+    // constants encoded as floating-point
+    __m512i magic_i_lo   = _mm512_set1_epi64x(0x4330000000000000); // 2^52
```

Comment:
This block is broken, there are no intrinsics `_mm512_set1_epi64x` and `_mm512_blend_epi32`.

Reply:
nice catch, I just forget to remove that part of code, it should be just `_mm512_cvtepi64_pd()` since the minimum support are SKX features which enables `AVX512DQ`